webapp/app/modules/lists/lists.selectors.js

```diff
@@ -0,0 +1,26 @@
+import { createSelector } from 'reselect';
+import { List } from 'immutable';
+
+
+const selectListsDomain = state => state.get('lists');
+
+export const selectListsData = createSelector(
+  selectListsDomain, state => state
+    .get('data')
+    .map((item) => item.set('items', item.get('items').sortBy((item) => item.get('name'))))
```

Comment:
it should be easier to use update instead of set when the new value depends on the old one

Reply:
`
item.update('items', sortBy(compose(toLower, prop('name')))
`

Reply:
you can assign this sortBy to the const and use it below